content/docs/reference/encryption.md

```diff
@@ -15,20 +15,27 @@ weight: 202 # For single pages, lower is first.
 ---
 
 ## Definition
-Encryption is a means of achieving privacy. It is a part of cryptography which has the Greek origin meaning of "secret writing". The goal of encryption is that assuming that an unwanted third party is eavesdropping on an insecure channel, said person would not be able to comprehend the transmitted information. Decryption is the process of converted encrypted information into a comprehensible format. Encryption and Decryption algorithms are known as ciphers. Encryption uses a key which is a set of values that the cipher, as an algorithm, will operate on. Encryption and Decryption go back a long way with one of the most well known ciphers being the [Caesar Cipher](https://en.wikipedia.org/wiki/Caesar_cipher)
+Encryption is a means of achieving privacy. 
+It is a part of cryptography which has the Greek origin meaning of "secret writing". 
+The goal of encryption is to stop unwanted third parties from comprehending transmitted information. 
+Decryption is the process of converting encrypted information into a comprehensible format. 
+Encryption and Decryption algorithms are known as ciphers. Encryption uses a key which is a set of values that the cipher, as an algorithm, will operate on. 
+Encryption and Decryption go back a long way with one of the most well known ciphers being the [Caesar Cipher](https://en.wikipedia.org/wiki/Caesar_cipher)
 
-## How we do it   
-The @platform implements end-to-end encryption that is best illustrated with the following example: @alice wishes to share her phone number with her friend @bob. To do this, @alice, who is on her own device, prompts her own secondary server to direct phone @alice at her friend @bob's secondary server. From here, a shared key is generated for @bob (@bob:shared_key@alice).
+## How We Do It
```

Comment:
Let's change this to How it works at Atsign to be consistent

Reply:
See 687c45650ae63f082974fe55527c9e4d8816a1d6